apps/api-documenter/src/documenters/MarkdownDocumenter.ts

```diff
@@ -502,6 +503,11 @@ export class MarkdownDocumenter {
   private _writePackageOrNamespaceTables(output: DocSection, apiContainer: ApiPackage | ApiNamespace): void {
     const configuration: TSDocConfiguration = this._tsdocConfiguration;
 
+    const abstractClassesTable: DocTable = new DocTable({
+      configuration,
+      headerTitles: ['Abstract Class', 'Description']
+    });
```

Comment:
I'd probably prefer having a "Modifiers" column instead of a separate section for alignment with other API items that have modifiers. What do you think?

Reply:
I wrote that as a side note in the issue:
> ...(my preferred solution) add a dedicated "Abstract Classes" overview table, because in an API, abstract classes are quite something different from a concrete class, sometimes closer to an interface.

As an API designer, I think of abstract classes as a third category in between interfaces and classes. Like interfaces, they cannot be instantiated, but like (non-abstract) classes, you can subclass them.

|                | defines type | subclass | instantiate |
-----------------|--------------|----------|-------------|
| interface      | ✅          | ❌       | ❌         |
| abstract class | ✅          | ✅       | ❌         |
| class          | ✅          | ✅       | ✅         |